bigtable-hbase-parent/bigtable-hbase-integration-tests-common/pom.xml

```diff
@@ -33,7 +33,7 @@ limitations under the License.
         <dependency>
             <groupId>org.apache.hbase</groupId>
             <artifactId>hbase-client</artifactId>
-            <version>${hbase.version}</version>
+            <version>${hbase.version.2}</version>
```

Comment:
We can't do this.  This has to stay `hbase.version`.  You need to make all changes related to hbase 2 in the integration-tests project.